### ISSUE TITLE
Funannotate -remote antismash sits at waiting for results

### ISSUE THREAD

**Issue by mswyers** [nextgenusfs/funannotate#317]
**Are you using the latest release?**
If you are not using the latest release of funannotate, please upgrade, if bug persists then report here.

**Describe the bug**
Using the remote version of antismash fails to ever return results. Manually uploading the same file to the antismash web service works, and manually running through a local install of antismash also works.

I let this sit overnight and nothing changed. Manually checking the status it's always at "pending". Uploading the same file manually finishes within 30 minutes. Downloading and extracting those files, however, doesn't work nicely with the annotate function for some reason (something about the .gbk formatting?) - it always says no CDS features found.

**What command did you issue?**
```
$ funannotate remote antismash -i predict/ -m antismash -e mswyers@intactgenomics.com
```
**Logfiles**
Please provide relavent log files of the error. 
```
[08/15/19 12:13:21]: /home/linuxbrew/.linuxbrew/Cellar/funannotate/1.5.3/libexec/bin/funannotate-remote.py -i predict/ -m antismash -e mswyers@intactgenomics.com

[08/15/19 12:13:21]: OS: linux2, 8 cores, ~ 33 GB RAM. Python: 2.7.15
[08/15/19 12:13:22]: Running funannotate v1.5.3
[08/15/19 12:13:22]: Output directory predict already exists, will use any existing data.  If this is not what you want, exit, and provide a unique name for output folder
[08/15/19 12:13:22]: Parsing input files
[08/15/19 12:13:22]: Generating files from predict/predict_results/Tolypocladium.gbk
[08/15/19 12:13:29]: Connecting to antiSMASH fungi v5 webserver
[08/15/19 12:13:29]: Queue Length: 0; Jobs Running: 2
[08/15/19 12:13:29]: PLEASE to not abuse the webserver, be considerate!
[08/15/19 12:13:29]: Uploading /home/seq/Desktop/TolyData/predict/predict_results/Tolypocladium.gbk to webserver
[08/15/19 12:28:17]: Waiting for results from job: fungi-32fe1e04-dec5-4494-b1b3-07cfa7e2fdac

```
**OS/Install Information**
 -  output of `funannotate check --show-versions`

```
$ funannotate check --show-versions
-------------------------------------------------------
Checking dependencies for funannotate v1.5.3
-------------------------------------------------------
You are running Python v 2.7.15. Now checking python packages...
biopython: 1.74
goatools: 0.9.7
matplotlib: 2.2.4
natsort: 6.0.0
numpy: 1.16.4
pandas: 0.24.2
psutil: 5.6.3
requests: 2.22.0
scikit-learn: 0.20.4
scipy: 1.2.2
seaborn: 0.9.0
All 11 python packages installed


You are running Perl v 5.030000. Now checking perl modules...
Bio::Perl: 1.007002
Carp: 1.50
Clone: 0.43
DBD::SQLite: 1.64
DBD::mysql: 4.050
DBI: 1.642
DB_File: 1.843
Data::Dumper: 2.174
File::Basename: 2.85
File::Which: 1.23
Getopt::Long: 2.51
Hash::Merge: 0.300
JSON: 4.02
LWP::UserAgent: 6.39
Logger::Simple: 2.0
POSIX: 1.88
Parallel::ForkManager: 2.02
Pod::Usage: 1.69
Scalar::Util::Numeric: 0.40
Storable: 3.15
Text::Soundex: 3.05
Thread::Queue: 3.13
Tie::File: 1.02
URI::Escape: 3.31
YAML: 1.29
threads: 2.22
threads::shared: 1.6
All 27 Perl modules installed


Checking external dependencies...
Traceback (most recent call last):
  File "/home/seq/emapper.py", line 19, in <module>
    from eggnogmapper.common import *
ImportError: No module named eggnogmapper.common
Soundex.c: loadable library and perl binaries are mismatched (got handshake key 0xce00080, needed 0xcd00080)
diamond v0.9.25.126 | by Benjamin Buchfink <buchfink@gmail.com>
Licensed under the GNU GPL <https://www.gnu.org/licenses/gpl.txt>
Check http://github.com/bbuchfink/diamond for updates.

RepeatModeler: RepeatModeler 1.0.8
Trinity: 2.8.3
augustus: 3.3.2
bamtools: bamtools 2.5.1
bedtools: bedtools v2.28.0
blat: BLAT v36
diamond: diamond 0.9.25
ete3: 3.1.1
exonerate: exonerate 2.4.0
fasta: no way to determine
gmap: 2019-06-10
gmes_petap.pl: 4.38
hisat2: 2.1.0
hmmscan: HMMER 3.2.1 (June 2018)
hmmsearch: HMMER 3.2.1 (June 2018)
java: 11.0.1
kallisto: 0.46.0
mafft: v7.407 (2018/Jul/23)
makeblastdb: makeblastdb 2.9.0+
minimap2: 2.17-r941
nucmer: 3.1
pslCDnaFilter: no way to determine
rmblastn: rmblastn 2.2.27+
samtools: samtools 1.9
stringtie: 1.3.6
tRNAscan-SE: 2.0.3 (April 2019)
tbl2asn: unknown, likely 25.3
tblastn: tblastn 2.9.0+
trimal: trimAl v1.4.rev15 build[2013-12-17]
	ERROR: CodingQuarry not installed
	ERROR: RepeatMasker not installed
	ERROR: emapper.py not installed
Checking Environmental Variables...
$FUNANNOTATE_DB=/home/seq/fundb
$PASAHOME=/home/seq/PASApipeline-v2.3.3
$TRINITYHOME=/home/linuxbrew/.linuxbrew/opt/trinity
$EVM_HOME=/home/linuxbrew/.linuxbrew/opt/evidencemodeler
$AUGUSTUS_CONFIG_PATH=/home/linuxbrew/.linuxbrew/opt/augustus/config
$GENEMARK_PATH=/home/seq/gmes_petap
$BAMTOOLS_PATH=/home/linuxbrew/.linuxbrew/opt/bamtools
All 7 environmental variables are set
```

**Comment by nextgenusfs** [comment]
possible that the antiSMASH api has changed

**Comment by nextgenusfs** [comment]
antismash remote search fixed in v1.7.0
